lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/NodeDAOImpl.java

```diff
@@ -2034,5 +2039,29 @@ public String getNodeName(String nodeId) {
 			throw new NotFoundException();
 		}
 	}
+	
+
+	@Override
+	public Long findFirstBoundJsonSchema(Long nodeId) {
```

Comment:
We might want to expose the max depth as a parameter so we can test it and set the default at the manager level